.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -62,41 +166,49 @@ jobs:
         with:
           access_token: ${{ github.token }}
 
-      - uses: actions/checkout@v2
-
-      - uses: docker/setup-qemu-action@v1
-        name: Set up QEMU
+      - uses: actions/checkout@v3
 
-      - uses: actions/setup-python@v2
-        name: Install Python
+      - name: Restoring cached dependencies
+        id: kokkos-cache
+        uses: actions/cache@v3
         with:
-          python-version: '3.7'
+          path: ${{ github.workspace }}/KOKKOS/${{ matrix.exec_model }}
+          key: ${{ matrix.container_img }}-kokkos${{ matrix.kokkos_version }}-${{ matrix.exec_model }}
+
+      - name: Copy cached libraries
+        run: |
+          cp -rf ${{ github.workspace }}/KOKKOS/${{ matrix.exec_model }}/* ./
 
       - name: Install cibuildwheel
-        run: python -m pip install cibuildwheel==2.3.0
+        run: python3 -m pip install cibuildwheel==2.8.0
+
+      - uses: docker/setup-qemu-action@v2
+        name: Set up QEMU
 
       - name: Build wheels
-        run: python -m cibuildwheel --output-dir wheelhouse
         env:
+          CIBW_ARCHS_LINUX: ${{matrix.arch}}
+
           CIBW_BUILD: ${{ matrix.cibw_build }}
+
           CIBW_SKIP: "*-musllinux*"
 
           # Python build settings
           CIBW_BEFORE_BUILD: |
-            pip install pybind11 ninja cmake && yum install -y gcc gcc-c++
+            cat /etc/yum.conf | sed "s/\[main\]/\[main\]\ntimeout=5/g" > /etc/yum.conf
```

Comment:
Do we need this line?

Reply:
This was added as the CI image would spend ~30s trying to hit remote URLs that were no longer in service. This line ensures that timeout is 5s only, and saved us a lot of deadtime over the course of setup and execution.